python/tk_desktop/rpc.py

```diff
@@ -360,9 +367,10 @@ def close(self):
         logger.debug("server setting flag to stop")
         self.server.close()
         self._is_closed = True
-        if sys.platform == "win32":
-            # The accept call blocks until a client is available.
-            # Unfortunately closing the connection on Windows does not
+
+        if six.PY2 and sys.platform == "win32":
```

Comment:
@mathurf, do you recall a bug that happened when you were validating SG-14617? This got added at that point and doesn't look natural. It looks like a patch to fix an issue encountered while working on that feature.

Reply:
Thanks for the ticket id. I took a look and all issues I mentioned in that ticket are not happening.